os/kernel/debug/Make.defs

```diff
@@ -33,5 +33,9 @@ ifeq ($(CONFIG_ENABLE_STACKMONITOR)$(CONFIG_DEBUG),yy)
 CSRCS += stackinfo_save_terminated.c
 endif
 
+ifeq ($(CONFIG_SE),y)
+CSRCS += secure_log.c
```

Comment:
is it secure element's default operation?
I think, It is operation using secure element.
so How about add new config?